projects/core/src/occ/adapters/product/converters/product-name-normalizer.ts

```diff
@@ -1,25 +1,38 @@
 import { Injectable } from '@angular/core';
+import { Product } from '../../../../model/product.model';
+import { Converter } from '../../../../util/converter.service';
 import { OccConfig } from '../../../config/occ-config';
 import { Occ } from '../../../occ-models/occ.models';
-import { Converter } from '../../../../util/converter.service';
-import { Product } from '../../../../model/product.model';
 
 @Injectable({ providedIn: 'root' })
 export class ProductNameNormalizer implements Converter<Occ.Product, Product> {
   constructor(protected config: OccConfig) {}
 
   convert(source: Occ.Product, target?: Product): Product {
-    if (target === undefined) {
-      target = { ...(source as any) };
-    }
+    target = target ?? { ...((source as any) as Partial<Product>) };
```

Comment:
According to our TS expert, @marlass, we should avoid using `any`, and use `unknown` instead.